components/sup/src/manager/service/hooks.rs

```diff
@@ -673,7 +660,7 @@ mod tests {
         ////////////////////////////////////////////////////////////////////////
 
         let hook_table = HookTable::load(&service_group, &template_path, &hooks_path);
-        assert_eq!(hook_table.compile(&service_group, &ctx), true);
+        assert_eq!(hook_table.compile(&service_group, &ctx).changed(), true);
```

Comment:
I know this was the existing code, but as long as we're modifying, I think
```suggestion
        assert!(hook_table.compile(&service_group, &ctx).changed());
```
reads a bit clearer. Similarly elsewhere.